webapp/components/user_settings/user_settings_notifications.jsx

```diff
@@ -182,6 +185,8 @@ export default class NotificationsTab extends React.Component {
         if (!Utils.areObjectsEqual(newState, this.state)) {
             this.setState(newState);
         }
+
+        this.setState({isSaving: false});
```

Comment:
Why is this updated here?

Reply:
Because updateState() is the method which is called when the notifications settings are successfully updated.